diff --git a/menu/cbs/menu_cbs_ok.c b/menu/cbs/menu_cbs_ok.c
index 6a16485805..ad077d5cc6 100644
--- a/menu/cbs/menu_cbs_ok.c
+++ b/menu/cbs/menu_cbs_ok.c
@@ -250,7 +250,6 @@ static int (funcname)(const char *path, const char *label, unsigned type, size_t
 #ifdef HAVE_LAKKA
 static char *lakka_get_project(void)
 {
-#ifndef HAVE_LAKKA_CANARY
    size_t len;
    static char lakka_project[128];
    FILE *command_file = popen("cat /etc/release | cut -d - -f 1", "r");
@@ -263,9 +262,6 @@ static char *lakka_get_project(void)
 
    pclose(command_file);
    return lakka_project;
-#else
-   return "/";
-#endif
 }
 #endif
 #endif
